index.html

```diff
@@ -343,8 +343,11 @@
           <li>If the <a>browsing context</a> of the script calling the
           constructor is a <a>nested browsing context</a> whose origin is
           different from the <a>top-level browsing context</a>'s origin and the
-          nested browsing context is not <a>allowed to make payment
-          requests</a>, then <a>throw</a> a <a>SecurityError</a>.
+          nested <a>browsing context</a>'s <a>browsing context container</a> is
+          an <a><code>iframe</code></a> element whose <a>node document</a> is
```

Comment:
Do we plan to allow nested `frame`/`object`/`embed` to use `PaymentRequest`. I thought not.